pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -225,7 +225,9 @@ func (k *KogitoServiceSpec) AddResourceLimit(name, value string) {
 func (k *KogitoServiceSpec) GetDeploymentLabels() map[string]string { return k.DeploymentLabels }
 
 // SetDeploymentLabels ...
-func (k *KogitoServiceSpec) SetDeploymentLabels(labels map[string]string) { k.DeploymentLabels = labels }
+func (k *KogitoServiceSpec) SetDeploymentLabels(labels map[string]string) {
+	k.DeploymentLabels = labels
+}
```

Comment:
Please let this on one single line for better readability

Reply:
Changed it back. Thanks.